err_stash.py

```diff
@@ -672,6 +749,55 @@ def stash_token(self, msg, args):
             self.save_user_settings(user, settings)
             return "Token saved."
 
+    @botcmd(name="delete-branch", split_args_with=None)
+    def delete_branch(self, msg, args):
+        """Declines given branch's PR and delete remaining branches"""
+        user = msg.frm.nick
+        if len(args) > 1:
+            return f"Unknown arguments {args[1:]}, please pass only the branch name to be deleted."
+        branch_to_delete = args[0]
+        settings = self.load_user_settings(user, additional_settings={"delete-branches": ""})
+        if not settings["token"]:
```

Comment:
Shouldn't you check for GitHub's token too?

Reply:
Yep, missed that, fixing